core/src/main/scala/org/apache/spark/ui/jobs/JobsTab.scala

```diff
@@ -37,7 +38,7 @@ private[ui] class JobsTab(parent: SparkUI, store: AppStatusStore)
     store
       .environmentInfo()
       .sparkProperties
-      .contains(("spark.scheduler.mode", SchedulingMode.FAIR.toString))
+      .contains((SCHEDULER_MODE.key, SchedulingMode.FAIR.toString))
```

Comment:
Do we want to just `get` the value here and check it? I'm not sure why it was written to scan the properties for a key-value pair.

Reply:
I think that this code intends to check a key-value pair in `sparkProperties` that is `Seq((String, String))`. This does not seem to get a value in config property.

Reply:
I see, .sparkProperties is a `Seq[(String, String)]`. Elsewhere it's converted to a Map to query it, but that seems like overkill